printer_mods/PurchenZuPoden/V0SKIRT_W_DISPLAY/README.md

```diff
@@ -0,0 +1,24 @@
+This is my Skirt Mod to add a FYSETC-Mini-12864-Panel Display, Version 2.1 to the V0
+ 
+![alt text](https://github.com/PurchenZuPoden/VoronUsers/blob/master/printer_mods/PurchenZuPoden/V0SKIRT_W_DISPLAY/03.JPG?raw=true)
+![alt text](https://github.com/PurchenZuPoden/VoronUsers/blob/master/printer_mods/PurchenZuPoden/V0SKIRT_W_DISPLAY/04.JPG?raw=true)
+![alt text](https://github.com/PurchenZuPoden/VoronUsers/blob/master/printer_mods/PurchenZuPoden/V0SKIRT_W_DISPLAY/02.JPG?raw=true)
+![alt text](https://github.com/PurchenZuPoden/VoronUsers/blob/master/printer_mods/PurchenZuPoden/V0SKIRT_W_DISPLAY/01.JPG?raw=true)
+
+- The cut-out in the Back Skirt is for a 
+Schaffner FN282-2-06 DC Power Connector with filter:
+https://uk.rs-online.com/web/p/iec-filters/1704958/
+
+- The back skirt is also designed for a Raspberry Pi 4
+
+- All parts are also available as a step file if you need to modify or adapt them
+
+- Add the display chapter to your printer.cfg in klipper (<a href="https://github.com/PurchenZuPoden/VoronUsers/blob/master/printer_mods/PurchenZuPoden/V0SKIRT_W_DISPLAY/Display_Config.txt" target="_blank">Display_Config.txt</a>)
+
+- Use the attached schematic for the wiring (<a href="https://raw.githubusercontent.com/PurchenZuPoden/VoronUsers/master/printer_mods/PurchenZuPoden/V0SKIRT_W_DISPLAY/Display_wiring.png" target="_blank">Display_wiring.png</a>)
+
+- The small bottom acrylic panel in the back is not used
+
+- The acrylic backpanel is cut to a height of 240mm
+
+- The Display Case (case.stl) needs a little bit of support 
```

Comment:
We cannot merge STLs that require support, all parts must be printable as is, just like other Voron STLs.  Please find a way to design this without requiring support.  If support absolutely cannot be avoided, then it must be built-in to the STL, with instructions to remove it after printing.

Reply:
I really don't know how to fix this.

Reply:
How does the display attach to your case?  Maybe you can break the case into two parts, the main case and a front ring?  Would the front ring need something to hold it in place, or would the display itself hold it?  You might need a way to snap the ring in if the display won't hold it.
<img width="678" alt="Screen Shot 2021-05-02 at 09 44 46" src="https://user-images.githubusercontent.com/69389593/116815444-98c62d00-ab2b-11eb-9849-738d288770b0.png">
<img width="680" alt="Screen Shot 2021-05-02 at 09 45 22" src="https://user-images.githubusercontent.com/69389593/116815449-9bc11d80-ab2b-11eb-9385-811ffb4b6ee5.png">

For example, look at how this display is attached on V2.4: https://github.com/VoronDesign/Voron-2/tree/Voron2.4/STLs/VORON2.4/Electronics_Compartment/LCD_Module